Assignment1/Assignment1_QNo8/SolAssignmentNo1_QNo8.rb

```diff
@@ -0,0 +1,24 @@
+# => Write a method which will take first name and
+# => last name as two arguments and return the fullname.
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def do_merge_name(fname,lname)
+  # Merging names with a space in between
+  puts "The required merged name is with white space in between #{fname} #{lname}"
```

Comment:
@HrisikeshSinghaChowdhury I could not see any code that will return the merged name. Can you please update the method